src/main/webapp/components/template/slider/slider.html

```diff
@@ -0,0 +1,3 @@
+<div id="slider"></div>
```

Comment:
Riiight. I did not know how to grab it from document except to use id. I will work with some other API to grab the div component from the directive!